pubspec.yaml

```diff
@@ -35,6 +35,7 @@ dependencies:
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
+  provider: ^6.0.0
```

Comment:
Remove this import if you dont need it

Reply:
You're right. I forgot to delete it after i changed what you asked me in the previous comments. Fixed it!